valispace/__init__.py

```diff
@@ -100,8 +100,8 @@ def all_vali_names(self):
 
 		return valinames.json()
 
-	def get_vali(self, id=None, name=None):
-		""" Returns the correct Vali. Input can be id or name """
+	def get_vali(self, id=None, name=None, workspace_id=None, project_id=None, parent_id=None, parent_name=None, tag_id=None, tag_name=None, valis_marked_as_impacted=None):
+		""" Returns the correct Vali(s) """
```

Comment:
Indicate here the return format (dictionary) and the format of the input arguments (integers for the IDs, strings for names).